processing/src/test/java/io/druid/query/topn/TopNQueryTest.java

```diff
@@ -74,4 +76,22 @@ public void testQuerySerialization() throws IOException
     Assert.assertEquals(query, serdeQuery);
   }
 
+  @Test
+  public void testQuerySerdeWithAlphaNumericTopNMetricSpec() throws IOException{
+    TopNQuery query = new TopNQueryBuilder()
+        .dataSource(QueryRunnerTestHelper.dataSource)
```

Comment:
I will change QueryRunnerTestHelper.dataSource to only "dataSource" as in other test.